src/components/AssetDialog.tsx

```diff
@@ -1,11 +1,13 @@
-import { Brand } from '@agoric/ertp';
 import { useAtomValue } from 'jotai';
 
 import AssetListItem from 'components/AssetListItem';
 import ListItem from 'components/ListItem';
 import SkeletonListItem from 'components/SkeletonListItem';
 import { displayFunctionsAtom } from 'store/app';
 
+// Ambient
+import '@agoric/ertp/src/types.js';
```

Comment:
Is it appropriate to use `import type` here?

Reply:
Ideally but the problem is the types needed are defined ambiently instead of in a module. We'll get there eventually but it's a lot more work https://github.com/Agoric/agoric-sdk/issues/4560